pkg/apis/admissionregistration/types.go

```diff
@@ -123,8 +124,78 @@ type ValidatingAdmissionPolicy struct {
 	metav1.ObjectMeta
 	// Specification of the desired behavior of the ValidatingAdmissionPolicy.
 	Spec ValidatingAdmissionPolicySpec
+	// The status of the ValidatingAdmissionPolicy, including warnings that are useful to determine if the policy
+	// behaves in the expected way.
+	// Populated by the system.
+	// Read-only.
+	// +optional
+	Status ValidatingAdmissionPolicyStatus
+}
+
+// ValidatingAdmissionPolicyStatus represents the status of an admission validation policy.
+type ValidatingAdmissionPolicyStatus struct {
+	// The generation observed by the controller.
+	// +optional
+	ObservedGeneration int64
+	// The results of type checking for each expression.
+	// Presence of this field indicates the completion of the type checking.
+	// +optional
+	TypeChecking *TypeChecking
+	// The conditions represent the latest available observations of a policy's current state.
+	// +optional
+	// +listType=map
+	// +listMapKey=type
+	Conditions []ValidatingAdmissionPolicyCondition
+}
+
+// ValidatingAdmissionPolicyCondition describes the state of an admission validation policy at a certain point.
+type ValidatingAdmissionPolicyCondition struct {
+	// The type of deployment condition.
+	Type ValidatingAdmissionPolicyConditionType
+	// The status of the condition, one of True, False, Unknown.
+	Status api.ConditionStatus
+	// The last time this condition was updated.
+	LastUpdateTime metav1.Time
```

Comment:
LastUpdateTime specifically is an anti-pattern we want to avoid in new condition fields. we only want to have to modify timestamps on transitions/changes

Reply:
Changing to use `metav1.Condition`. This field no longer exist. (resolved)